examples/vars.yml

```diff
@@ -0,0 +1,25 @@
+postgres_password: "{{ lookup('password', 'inventory/host_vars/{{domain}}/passwords/postgres.psk chars=ascii_letters,digits') }}"
+
+# You can set any pict-rs environmental variables here. They will populate the templates/docker-compose.yml file.
+# https://git.asonix.dog/asonix/pict-rs
+pictrs_env_vars:
+  - PICTRS__SERVER__API_KEY: "{{ postgres_password }}"
+  - PICTRS__MEDIA__VIDEO_CODEC: vp9
+  - PICTRS__MEDIA__GIF__MAX_WIDTH: 256
+  - PICTRS__MEDIA__GIF__MAX_HEIGHT: 256
+  - PICTRS__MEDIA__GIF__MAX_AREA: 65536
+  - PICTRS__MEDIA__GIF__MAX_FRAME_COUNT: 400
+  - PICTRS_OPENTELEMETRY_URL: http://otel:4137
+  - RUST_LOG: debug
+  - RUST_BACKTRACE: full
+#  - PICTRS__STORE__TYPE: object_storage 
```

Comment:
Thanks a ton for adding these as comments, its going to be really helpful to people who want to use object storage right away.